### PR TITLE
Add gar-t5 topics to Pyserini's prebuilt topics

### PR DESCRIPTION
**Pull request by ArthurChen189**
@ronakice

### CODE REVIEW

**Review by ronakice — APPROVED** [review]
LGTM!

File: pyserini/search/_base.py [hunk 1]
```diff
@@ -129,6 +129,14 @@
...
     'dpr-squad-test': JTopics.DPR_SQUAD_TEST,
     'dpr-curated-test': JTopics.DPR_CURATED_TEST,
+    'gar-t5-nq-test-answers': JTopics.GART5_NQ_TEST_ANSWERS,
+    'gar-t5-nq-test-titles': JTopics.GART5_NQ_TEST_TITLES,
+    'gar-t5-nq-test-sentences': JTopics.GART5_NQ_TEST_SENTENCES,
+    'gar-t5-nq-test-all': JTopics.GART5_NQ_TEST_ALL,
+    'gar-t5-dpr-trivia-test-answers': JTopics.GART5_DPR_TRIVIA_TEST_ANSWERS,
+    'gar-t5-dpr-trivia-test-titles': JTopics.GART5_DPR_TRIVIA_TEST_TITLES,
+    'gar-t5-dpr-trivia-test-sentences': JTopics.GART5_DPR_TRIVIA_TEST_SENTENCES,
+    'gar-t5-dpr-trivia-test-all': JTopics.GART5_DPR_TRIVIA_TEST_ALL,
```
Comment (ronakice) [inline pyserini/search/_base.py:139]: Actually, what about `gar-t5-answers` followed by the `nq-test` and so on, this is motivated by preserving the original naming schemes.

File: docs/experiments-gar-t5.md [hunk 2]
```diff
@@ -3,33 +3,11 @@
...
-  --titles <optional> \
-  --answers <optional>
-```
-
 ## GAR-T5 enhanced retrieval evaluation
-To evaluate the augmented queries, we need to concatenate and convert them into .tsv format for us to run BM25-search on Pyserini, which is then converted to .json format as required for evaluation.
-
-Without specifying the output path, the default output will be an `augmented_topics.tsv` file in the working directory.
-
-Once we have the tsv file, we can proceed to run search and evaluation
```
Comment (ronakice) [inline docs/experiments-gar-t5.md:28]: Perhaps include the first half of this in the end of the README file? Perhaps

## Method 2: Interacting with GAR-T5 Predictions

If you want to generate the pre-built topics yourself from the GAR-T5 Predictions, ... [enter whatever you have in the `Get the download...`

File: docs/experiments-gar-t5.md [hunk 3]
```diff
@@ -3,33 +3,11 @@
...
 
 ```bash
 python -m pyserini.search \
-  --topics augmented_topics.tsv \
+  --topics <dpr-trivia, or nq>-test-gar-t5-<answers, titles, sentences, or all> \
```
Comment (ronakice) [inline docs/experiments-gar-t5.md:10]: no need the comma after dpr-trivia

**Review by ronakice — CHANGES_REQUESTED** [review]
2 minor fixes and I'll merge.

**Review by ronakice — APPROVED** [review]
LGTM! Merging

**Comment by lintool** [comment]
Sorry to chime in late, but @ArthurChen189 can we get some tests please? https://github.com/castorini/pyserini/blob/master/tests/test_load_qrels.py

**Pull request merged**